synapse/lib/ast.py

```diff
@@ -3821,7 +4128,7 @@ async def destfilt(destforms, node, path):
 
                 async for walknode in self.walkNodeEdges(runt, node, verb=verb):
 
-                    if not await destfilt(dest, walknode, path):
+                    if destfilt and not await destfilt(walknode, path, cmprvalu):
                         await asyncio.sleep(0)
```

Comment:
walkNodeEdges has already had a `sleep(0)` for the node being lifted ( via getNodeByBuid ) so this sleep should not be required?

Reply:
removed